packages/react-instantsearch-core/src/core/__tests__/createInstantSearchManager.js

```diff
@@ -615,6 +624,130 @@ describe('createInstantSearchManager', () => {
+            <Configure filters="brand:Samsung" />
+          </Index>
+
+          <Index indexId="instant_search_microsoft" indexName="instant_search">
+            <Configure filters="brand:Microsoft" />
+          </Index>
+        </InstantSearch>
+      );
+
+      const {
+        mainParameters,
+        derivedParameters,
+      } = wrapper.instance().aisManager.getSearchParameters();
+
+      expect(mainParameters).toEqual(
+        expect.objectContaining({
+          index: 'index1',
+        })
+      );
+
+      expect(derivedParameters).toEqual([
+        expect.objectContaining({
+          indexId: 'bestbuy',
+          parameters: expect.objectContaining({
+            index: 'bestbuy',
+          }),
+        }),
+        expect.objectContaining({
+          indexId: 'instant_search',
+          parameters: expect.objectContaining({
+            index: 'instant_search',
+          }),
+        }),
+        expect.objectContaining({
+          indexId: 'instant_search_apple',
+          parameters: expect.objectContaining({
+            index: 'instant_search',
+            filters: 'brand:Apple',
+          }),
+        }),
+        expect.objectContaining({
+          indexId: 'instant_search_samsung',
+          parameters: expect.objectContaining({
+            index: 'instant_search',
+            filters: 'brand:Samsung',
+          }),
+        }),
+        expect.objectContaining({
+          indexId: 'instant_search_microsoft',
+          parameters: expect.objectContaining({
+            index: 'instant_search',
+            filters: 'brand:Microsoft',
+          }),
+        }),
+      ]);
+    });
+
+    it('expects widgets main parameters and derived parameters to be correctly calculated with SortBy within a multi index context', () => {
+      const wrapper = mount(
+        <InstantSearch indexName="index1" searchClient={createSearchClient()}>
+          <Index indexName="categories">
+            <SortBy
+              defaultRefinement="bestbuy"
+              items={[
+                { value: 'categories', label: 'Categories' },
+                { value: 'bestbuy', label: 'Best buy' },
+              ]}
+            />
+          </Index>
+
+          <Index indexName="products">
+            <SortBy
+              defaultRefinement="brands"
+              items={[
+                { value: 'products', label: 'Products' },
+                { value: 'brands', label: 'Brands' },
+              ]}
+            />
+          </Index>
+        </InstantSearch>
+      );
+
+      const {
+        mainParameters,
+        derivedParameters,
+      } = wrapper.instance().aisManager.getSearchParameters();
+
+      expect(mainParameters).toEqual(
+        expect.objectContaining({
+          index: 'index1',
+        })
+      );
+
+      expect(derivedParameters).toHaveLength(2);
+
+      expect(derivedParameters).toEqual(
+        expect.arrayContaining([
```

Comment:
> https://github.com/algolia/react-instantsearch/pull/2570/commits/8f46d590f8a08948894b90b8f7d6dc76b2934118

It was not the only place where it was used.

Reply:
Oups, forgot the other one.